config/offices.yml

```diff
@@ -0,0 +1,27 @@
+# APP_RELEASE_STAGE
+defaults: &defaults
+  # Which office to send the fax to in the event the submitted
+  # zip code is not in our mapping
+  fallback_office: "union"
+
+  # Because YML inheritance doesn't work well with nested hashes,
+  # We are relying on matching the list of offices per-environment with
+  # the list of zip codes a given office serves to reduce configuration duplication.
+  # This may be unnecessary?
```

Comment:
Makes sense -- don't think we *need* this comment as long as we have good tests for what's going on in here :)